cmd/external-repos/main.go

```diff
@@ -43,15 +47,33 @@ func main() {
 		log.Debug().Msg("Successfully loaded external repositories.")
 	} else if args[1] == "introspect" {
 		if len(args) < 3 {
-			log.Fatal().Msg("Usage:  ./external_repos introspect URL")
+			log.Panic().Err(err).Msg("Usage:  ./external_repos introspect [--force] URL")
+			os.Exit(1)
 		}
-		count, errors := external_repos.IntrospectUrl(args[2])
+		// TODO Quick option implemented, refactor to use some library
```

Comment:
dbmigrate handles flags with this: https://github.com/content-services/content-sources-backend/blob/main/cmd/dbmigrate/main.go#L52-L56

It's not a great library but it's simple